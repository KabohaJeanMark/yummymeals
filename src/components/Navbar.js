import { Link } from 'react-router-dom';

const Navbar = () => (
  <nav>
    <div>
      <Link to="/">yummyMeals</Link>
    </div>
  </nav>
);

export default Navbar;
