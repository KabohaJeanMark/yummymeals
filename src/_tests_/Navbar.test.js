import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import { render } from '@testing-library/react';
import Navbar from '../components/Navbar';

describe('Navbar component tests', () => {
  it('should match the snapshot', () => {
    const tree = renderer.create(
      <BrowserRouter>
        <Navbar />
      </BrowserRouter>,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should be defined', async () => {
    const nav = render(
      <BrowserRouter>
        <Navbar />
      </BrowserRouter>,
    );
    expect(nav).toBeDefined();
  });
});
