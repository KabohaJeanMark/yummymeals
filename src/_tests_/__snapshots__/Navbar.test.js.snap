// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar component tests should match the snapshot 1`] = `
<nav>
  <div>
    <a
      href="/"
      onClick={[Function]}
    >
      yummyMeals
    </a>
  </div>
</nav>
`;
